src/NosCore.Parser/Parsers/NpcMonsterParser.cs

```diff
@@ -112,7 +111,9 @@ public async Task InsertNpcMonstersAsync(string folder)
                 {nameof(NpcMonsterDto.DarkResistance), chunk => Convert.ToInt16(chunk["ATTRIB"][0][7])},
                 {nameof(NpcMonsterDto.MaxHp), chunk => Convert.ToInt32(chunk["HP/MP"][0][2]) + _basicHp[Level(chunk)]},
                 {nameof(NpcMonsterDto.MaxMp), chunk => Convert.ToInt32(chunk["HP/MP"][0][3]) + Convert.ToByte(chunk["RACE"][0][2]) == 0 ? _basicPrimaryMp[Level(chunk)] : _basicSecondaryMp[Level(chunk)]},
-                {nameof(NpcMonsterDto.Xp), chunk =>  Math.Abs(Convert.ToInt32(chunk["EXP"][0][2]) + _basicXp[Level(chunk)])},
+                {nameof(NpcMonsterDto.Xp), chunk =>  Level(chunk) >= 19 ? 
```

Comment:
Please make that a function

Reply:
Already done in local, I'm fixing jxp aswell that's why it's not pushed rn